.azure-pipelines/fluentJavaMgmt.yml

```diff
@@ -5,7 +5,7 @@ trigger:
   - master
 
 variables:
-  DefaultOptions: '--batch-mode -Dmaven.wagon.http.pool=false'
+  DefaultOptions: '-Dmaven.wagon.http.pool=false'
```

Comment:
Why did we need this change?

Reply:
If with batch mode, the maven job will always run from the parent pom directory. In this case, the suppressions.xml of cosmosdb will not work.  Thus, have to exclude batch mode in this case to run check style into each individual directory.